src/blockchain/Cargo.toml

```
@@ -27,4 +27,4 @@ name = "simple_node"
 
 [dev-dependencies]
 pretty_env_logger = "0.4.0"
-tokio = { version = "1.17.0", features = [ "macros", "rt-multi-thread", "io-std" ] }
+tokio = { version = "1.17.0", features = [ "macros", "rt-multi-thread", "io-std" ] }
```

Should be a newline here
```suggestion
tokio = { version = "1.17.0", features = [ "macros", "rt-multi-thread", "io-std" ] }

```